gateway/1.5/modules/ROOT/pages/_partials/product-spotlight-video.html

```diff
@@ -0,0 +1 @@
+<script type='text/javascript' async src='https://play.vidyard.com/embed/v4.js' data-playbackurl='play.vidyard.com'></script><img style='margin: auto; display: block; width: 100%; 'class='vidyard-player-embed' src='https://play.vidyard.com/3LncCho6hfvbLMKbg1AM28.jpg' data-height='540' data-width='960' data-width='auto' data-controller='hubs' data-action='show' data-uuid='3LncCho6hfvbLMKbg1AM28' data-type='inline' />
```

Comment:
Are the values of `data-height` and `data-width` standard across all our videos? Or are they unique to this video? Asking because if they're unique to this video, does it need to be tested on various screen resolutions? Like what happens on mobile?

Reply:
Hmm, this is all autogenerated. Beth doesn't mention it in the guide. 

data-height='540' data-width='960' is a 9:16 aspect ratio which is what youtube uses. So, I don't think we need to worry about this.